<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Friends</class>
 <widget class="QDialog" name="Friends">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget{
	background-color: #343A40;
	border: 1px #343A40;
}</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="Picture">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-10</y>
       <width>75</width>
       <height>75</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="HomeButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CalendarButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="FriendsButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Inicio">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>5</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: #343A40;
	border: 0px;
	padding: 5px;
	color: #D00000
}</string>
     </property>
     <property name="text">
      <string>Amigos</string>
     </property>
    </widget>
    <widget class="QLabel" name="Nombre">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>150</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: #343A40;
	border: 0px;
	color: #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Codigo">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>150</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	background-color: #343A40;
	border: 0px;
	color: #D00000;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>531</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>NÃºmero de solicitudes pendientes:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel{
	background-color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="EnviarSolicitud">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Enviar solicitud de amistad:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="CancelarSolicitud">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Cancelar solicitud de amistad:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	vackground-color: #FFFFFF;
	color: #3A86FF;
}</string>
   </property>
   <property name="text">
    <string>Personas que quizas conoscas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
